packages/react-tabs/src/components/TabList/useTabListStyles.ts

```diff
@@ -10,22 +17,115 @@ const useStyles = makeStyles({
   root: {
     display: 'flex',
     flexDirection: 'row',
+    position: 'relative',
   },
   vertical: {
     flexDirection: 'column',
   },
 });
 
+/**
+ * Styles for the selection indicator slot when horizontal.
+ */
+const useHorizontalIndicatorStyles = makeStyles({
```

Comment:
possible todo for later: horizontal and vertical styles have a ton in common. could be combined with horizontal/vertical being state just like small/animated

Reply:
The problem is that one is :before and one is :after.  I couldn't figure out a way to re-use common styles for both tags.  I didn't want to rely on the content being unset and applying a bunch of dummy styles to both.  Interested in hearing ways to do this :)